README.md

```diff
@@ -1,10 +1,22 @@
-# US Police Shootings
-**Content**
+# Project Title
 
-It contains basic data about people like their name, age, gender and race. 
-Along with it, is the shooting/killing information, like date of event, where it happened? How they were shot? 
-Did they attack? Were they holding weapons? Did they show any mental illness? Was the policeman wearing a camera/was the incident recorded? 
-Did the suspect flee? 
-Apart from that, a category column holds type of weapon used by the suspect
+An Analysis on Police Shootings to Predict Unjustified Shots
 
-[Here is a link to the dataset](https://www.kaggle.com/ahsen1330/us-police-shootings)
+## Motivation
+
+The death of George Floyd in the hands of the police sparked massive protests across different cities in America and the world. Caught on camera, the incident showed a subdued George Floyd begging to breathe as a police man placed his knee to his neck for several minutes. This caused outrage particularly in the black community where there is a distrust for the police. The Black Lives Matter (BLM) movement was at the forefront of the protests, in recent years, the BLM has gained popularity for their activism against police brutality against Black-Americans. 
+
+The protests were mostly peaceful, however, videos were littered all over twitter showing the police firing rubber bullets, stun guns and tear gas at protesters and journalists to intimidate them. This shocked many Americans who were for the first time witnessing their police’s heavy-handedness in dealing with peaceful protesters.
+
+In recent years, there have been numerous cases involving the fatal shooting of Black Americans by the police. In majority of these cases, the police officers involved in the shooting were let off with a slap on the wrist leaving the victims’ families unsatisfied with the ruling. 
+
+Based on this, we decided to conduct an analysis of police shooting incidents in America from 2015 to date, to ascertain if the shootings were justified in a bid to help victims and their families seeking justice to get it. To achieve this, we performed clustering technique on our data to discover patterns about police shootings that could help the victims and their families in their lawsuit against the police.
```

Comment:
We ended up using a Supervised learning approach. Can you please edit it, and change that line to something like (just a rough clue): To achieve this, we used a supervised learning algorithm to predict the probability of an unjustified shot. Cases with probabilities equal to or higher than 60% are most likely unjust. Hence, this approach could help the victims or their families and governing bodies in their lawsuit against such Police officer(s).

Reply:
Weldon @ForChune. It’s a nice READ ME and with the little adjustment spotted by @sharonibejih I think we’d be good to go.

Reply:
Thanks for the review! @sharonibejih @Jahfaith.  I've made some modifications to the second to the last paragraph based on your feedback.